Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rust:1.48 as builder
+
+RUN USER=root cargo new --bin lychee
+WORKDIR /lychee
+COPY Cargo.toml Cargo.toml
+RUN cargo build --release
+RUN rm src/*.rs
+
+ADD . ./
+
+RUN rm ./target/release/deps/lychee*
+RUN cargo build --release
+
+
+FROM debian:buster-slim
```

Comment:
Worth separating the stages with a comment

Reply:
True. Added.